resource-timing/resources/iframe-TAO-crossorigin-port.sub.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <script>
+let client = new XMLHttpRequest,
+    url = '{{location[scheme]}}://{{host]}}:{{ports[http][1]}}/resource-timing/resources/TAOResponse.py?tao=origin_port';
+    client.open('GET', url);
+    client.onload =(()=>{
+        setTimeout(()=>{
```

Comment:
Can we use PerformanceObserver here instead of relying on setTimeout?

Reply:
yeah, modernized the whole thing